x-pack/plugins/cases/common/api/connectors/swimlane.ts

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import * as rt from 'io-ts';
+export const SwimlaneUnmappedFieldsRT = rt.type({
+  alertSource: rt.union([rt.string, rt.null]),
+  caseId: rt.union([rt.string, rt.null]),
+  caseName: rt.union([rt.string, rt.null]),
+  severity: rt.union([rt.string, rt.null]),
+});
+// New fields should also be added at: x-pack/plugins/cases/server/connectors/case/schema.ts
+export const SwimlaneFieldsRT = rt.intersection([
```

Comment:
It seems that this is not used anywhere. Also the schema in `x-pack/plugins/cases/server/connectors/case/schema.ts` differes from this type.